src/components/PageHeader.vue

```diff
@@ -1,37 +1,68 @@
 <template>
-  <div class="header">
-    <a href="https://geoconnex.us" class="logo-link">
-      <img
-        src="@/assets/logo-geoconnex.png"
-        alt="Geoconnex Logo"
-        title="geoconnex.us"
-        class="logo-geoconnex"
-      />
-    </a>
+  <div class="header pa-6">
+    <div class="logo-container">
+      <a href="https://geoconnex.us">
+        <img src="@/assets/logo-geoconnex.png" alt="Geoconnex Logo" title="geoconnex.us" class="logo-geoconnex" />
+      </a>
+    </div>
+    <v-icon @click="toggleTheme" icon="mdi-theme-light-dark" class="theme-toggle-icon"></v-icon>
   </div>
+
 </template>
 
+
+<script setup lang="ts">
+import { useTheme } from 'vuetify'
+
+const theme = useTheme()
+
+function toggleTheme() {
+  theme.global.name.value = theme.global.current.value.dark ? 'light' : 'dark'
+}
+</script>
+
+
 <style scoped>
 .header {
-  width: 100%;
-  background-color: #f0f0f0;
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
-  padding: 10px 20px;
-  margin-bottom: 40px;
   display: flex;
-  justify-content: center;
   align-items: center;
+  /* Vertically center items */
+  justify-content: space-between;
+  /* Space out the logo and the theme icon */
+  padding: 10px;
+
+  background-color: var(--v-theme-primary);
```

Comment:
This was how I was trying to refer to the primary colors from the theme. I can see this is a variable that is defined in the browser and I have a primary color in my theme but for some reason it is not working.